.github/workflows/ci.yml

```diff
@@ -51,4 +51,4 @@ jobs:
       run: cargo fmt --all -- --check
 
     - name: docs
-      run: cargo +nightly doc --no-deps
+      run: cargo clean && cargo +nightly doc --no-deps
```

Comment:
Maybe good to add that as a comment in the source.
Another option that might be preferable would be to pin to a specific version of nightly, so we don't roll forward every time we build this. 

I wonder why we're running nightly on this, I'm curious if the reason may have been addressed in a stable version since this was created. but that could be a question for a different time.

Reply:
Both good questions - mine was why this was still an issue if the bug was patched and closed 10 days ago 🙃 But yeah, I wanna play with this more and at a minimum leave a comment before moving forward. Good idea with dropping the +nightly and seeing what happens.

Reply:
Great news! Not only is the `cargo clean` not an issue, but we don't appear to need nightly to build them either. I cleaned this all up and we should be good to go.